__device-tests__/utils.js

```diff
@@ -5,6 +5,25 @@ import jimp from 'jimp';
 
 const { isAndroid } = e2eUtils;
 
+const SUPPORTED_GLOBAL_STYLES_BLOCKS = [
+	'core/paragraph',
+	'core/button',
+	'core/quote',
+	'core/pullquote',
+	'core/quote',
+	'core/search',
+];
+const SUPPORTED_GLOBAL_STYLES_ELEMENTS = [
+	'h1',
+	'h2',
+	'h3',
+	'h4',
+	'h5',
+	'h6',
+	'link',
+	'button',
+];
```

Comment:
Is there a way for us to have a single source of truth for this in the `WordPress/gutenberg` repository? I am fearful this will (possibly silently) become out of date and difficult to maintain. 

I briefly searched for a simple solution, but was unable to find one. 😞

Reply:
Sorry for the late reply!

> Is there a way for us to have a single source of truth for this in the WordPress/gutenberg repository? I am fearful this will (possibly silently) become out of date and difficult to maintain.

I think we could have it with the rest of the configuration related to global styles for mobile so it's all in the same place.

I can draft a PR with a possible solution this week 👍